test/suites/ACID.m

```diff
@@ -523,6 +523,7 @@
 function [stat] = logplot()
   stat.description = 'Test logscaled axes.';
   stat.unreliable = isMATLAB('>=', [8,4]); %FIXME: #590
+  stat.unreliable = stat.unreliable || isMATLAB('<=', [8,3]); %FIXME: #749 (Jenkins)
 
```

Comment:
Yes, indeed and aesthetically, I completely agree with your remark.

However, I kept those as separate lines, because that allows to distinguish the different causes of unreliability. That should allow us to trace those problems more directly.